src/Packetery/Module/Order/Form.php

```diff
@@ -134,4 +144,31 @@ public function setDefaults(
 			]
 		);
 	}
+
+	/**
+	 * Creating a form
+	 *
+	 * @param string $orderShippingCountry Country of destination.
+	 * @return Forms\Form
+	 */
+	public function changeCarrier( string $orderShippingCountry ): Forms\Form {
```

Comment:
Metoda, která vyrábí formulář se nemůže jmenovat "změň dopravce" ani "změna dopravce".
Na vyrábění formuláře máme dedikovanou formFactory, dejme tuto metodu tam.
Nechť její název začíná slovem create. Tato třída by měla zůstat beze změny.

Reply:
Upraveno.

Reply:
Prosím odstranit zbytečné úpravy třídy Form.

Reply:
Je to pryc.